docs/3.0.0-beta.x/getting-started/introduction.md

```diff
@@ -1,18 +1,80 @@
 
-When you're done getting started, we invite you to join our [community](https://strapi.io/community).
+### Manage content
+
+The Strapi's admin panel provide you an intuitive interface to create edit and delete your content.
+
+### Developer-friendly API
+
+Your content have to be displayed on your devices, an API is required. Strapi provide you an API that match your need. Fetch the data you want via a REST API or via the GraphQL endpoint.
+
+### Roles & permissions
+
+A User system is built in Strapi, that let you manage how can access to what.
+
+### Plugin system
+
+You want a new feature in you application, fetch a plugin in the market place and tada! You can install a GraphQL endpoint in one command.
+
+### Customization
+
+You can customize every single pice of your application. The architecture of Strapi let you customize your app to match your exact needs.
+
+Don't stay stuck by the techno you choose.
+
+## The Strapi Ecosystem
+
+### Code Source
+
+Strapi is an Open Source project, all the source code is under the MIT licence. The core project is in the [strapi/strapi](https://github.com/strapi/strapi) GitHub repository and you will find all other tools into the [Strapi](https://github.com/strapi) GitHub organization.
+
+### Marketplace
+
+In the marketplace section of the admin panel, you will only found official Strapi plugins. For now, the Strapi marketplace is not open to everyone but will be soon, stay tuned.
+
+But you already can develop yours and publish them on npm.
+
+### Roadmap
+
+Strapi is a community oriented project. We are trying to be the more transparent as possible to let you have a vision on the future of the project.
+Your insights are really important too to help to develop the project in the right direction.
+The [Roadmap](https://portal.productboard.com/strapi) is open, feel free to add your insights.
+
+### Tutorials and Guides
+
+To help you using Strapi, we and the community provide you a suite of resources to let you achieve your goals.
+You will find [tutorials](https://strapi.io/tutorials/) on the website and guides into the documentation.
+
+### Demo
+
+You can discover Strapi in few minutes by watching the demo video or by requesting a live demo via the form present in the [Demo](https://strapi.io/demo) page.
+
+### Blog
+
+For long-form news and announcements the best place to keep up to date with everything happening is our [blog](https://strapi.io/blog).
+
+### Social media
+
+We maintain a handful of social media accounts to keep in touch with the community directly and give them updates about what's new with Strapi. Here are the easiest ways to keep track of Strapi and everything hapening around the web:
+
+- ⭐️ [GitHub](https://github.com/strapi/strapi)
+- 🐧 [Twitter](https://twitter.com/strapijs)
+- 👩🏻‍💻 [Facebook](https://www.facebook.com/strapijs/)
+- 📸 [Instagram](https://www.instagram.com/strapijs/)
```

Comment:
and share updates about the project and company. Here are the easiest ways to keep track of Strapi and everything happening around the web:
- [reddit](https://www.reddit.com/r/Strapi/)
- [Linkedin](https://www.linkedin.com/company/strapi/)